src/data/reconstructing_datatypes.jl

```diff
@@ -308,9 +308,7 @@ function _resolve_type(rr::MappedRepr{T,DataTypeODR},
                        hasparams::Bool,
                        params) where T
     parts = split(mypath, '.')
-    modules = vcat([Main], collect(keys(Base.module_keys)))
-    unique!(modules)
-    for mod in modules
+    for mod in values(Base.loaded_modules)
```

Comment:
You should preferably use `Base.loaded_modules_array`, as that has been the stable and safe API for this for many versions (since JuliaLang/julia#23579), while this is neither safe nor stable

Reply:
Nice! I didn't know that function existed.
Thanks, I'll switch to that in the next release.